.github/workflows/sync-from-external-sources.yml

```diff
@@ -7,6 +7,8 @@ on:
 jobs:
   sync-emojis:
     runs-on: ubuntu-latest
+    # Skip in forks
+    if: github.repository == 'vector-im/element-android'
```

Comment:
Maybe you can also create an `env` variable containing the `vector-im/element-android` at the top of the file not to duplicate the repository name